tensorflow/lite/micro/tools/make/kissfft_download.sh

```diff
@@ -59,7 +59,7 @@ else
 
   pushd ${DOWNLOADED_KISSFFT_PATH} > /dev/null
   create_git_repo ./
-  apply_patch_to_folder ./ ../../kissfft.patch "TFLM patch"
+  apply_patch_to_folder ./ ../../../../../../../third_party/kissfft/kissfft.patch "TFLM patch"
```

Comment:
Kinda awkward with the multiple ../? I tried with the ${2} root but it didn't work.

Reply:
yeah, that is just what it is right now.